assets/content/pages/work-with-us/carouselContent.json

```diff
@@ -182,11 +182,11 @@
           "Let us know what you are looking for and we'll sync you with the right team."],
         "cta": {
           "text": "Visit BanklessDAO Podcasts",
-          "link": "https://www.bankless.community/listen",
+          "link": "/listen",
```

Comment:
I fixed this internal link. It didn't work when I added the `/bankless-website` prefix. :shrug:  not sure why.